packages/daemon/src/daemon-node-powers.js

```diff
@@ -310,6 +310,20 @@ export const makePowers = ({
     return fs.promises.readFile(path, 'utf-8');
   };
 
+  /**
+   * @param {string} path
+   */
+  const maybeReadFileText = async path =>
+    readFileText(path).catch(error => {
+      if (
+        error.message.startsWith('ENOENT: ') ||
+        error.message.startsWith('EISDIR: ')
```

Comment:
is there a subprop or code instead of inspecting the message?

Reply:
`error.code` should be available but my experiments with it did not pan out.